packages/SQLServerDataProvider/src/SQLServerDataProvider.ts

```diff
@@ -1883,14 +1883,14 @@ export class SQLServerDataProvider extends ProviderBase implements IEntityDataPr
             }
             else {
                 // got our field, create a SQL Query
-                let sql: string = `SELECT ${f.Name} FROM [${e.SchemaName}].[${e.BaseView}] WHERE `
+                let sql: string = `SELECT [${f.Name}] FROM [${e.SchemaName}].[${e.BaseView}] WHERE `
                 let where: string = '';
                 for (let pkv of CompositeKey.KeyValuePairs) {
                     const pk = e.PrimaryKeys.find(pk => pk.Name === pkv.FieldName);
```

Comment:
In this case, I did not change it to use `CodeName` since (afaict) the `FieldName` value from the CompositeKey class would generally be the actual column name (with spaces)